plugins/woocommerce/src/Internal/Admin/Logging/FileV2/File.php

```diff
@@ -430,6 +430,10 @@ protected function create(): bool {
 	 * @return bool
 	 */
 	public function write( string $text ): bool {
+		if ( $text === '' ) {
+			return true;
```

Comment:
```suggestion
			return false;
```

It seems more accurate for this to return false, rather than true, since no file write is occurring in this case.

Reply:
Yeah, I see the reasoning there. In this case, we have a cache for the results when you do a string search across multiple files, and that cache gets invalidated when a write is successful. We don't need the cache to get invalidated if no write actually happened, so I think false would be better in this case.